### DISCUSSION TITLE
Data Camp free 3 Months trials nut working. (Fix)

### DISCUSSION THREAD

**Discussion by LEARNERMUSTAFA** [community/community#75584] [GitHub Education]
### Hi everyone,

Hi, As github is giving Free 3-month individual subscription for students on datacamp but I got only for 1 week. Why? when I next time login on datacamp, it says your coupon has been expired or not valid. I'll appreciate you, for any kind of solution.
![Screenshot (111)](https://github.com/community/community/assets/149287596/43d7a632-af83-45df-9462-9bf15628c575)

**Comment by masondao** [comment]
Any updates on this?

**Comment by molbiologist85** [comment]
can somebody tell me how can I get data camp free for students?

**Comment by meznader** [comment]
same from me i was waiting to join data camp it say : 
This discount has expired or cannot be applied to your current plan.[Click here for more info.](https://support.datacamp.com/hc/en-us/articles/360012481334-Why-Am-I-Not-Eligible-for-the-Promo)

![image](https://github.com/community/community/assets/162059171/dcd7f231-dc92-47d9-9ed6-a5beb7f183b4)

**Comment by Nicotnn** [comment]
Has anyone gotten an update on this? Just encountered the same issue

**Comment by okimazzi** [comment]
hey guys, contact DataCamp support and show your institutional email and active student pack subscription ![image](https://github.com/user-attachments/assets/2c8cf4e7-ab46-44c9-b679-6c417b74b53e)

**Comment by Hamza-Elbk** [comment]
I tried to sign in before get Trial and its working
